client/my-sites/menus/index.js

```diff
@@ -1,18 +1,18 @@
 /**
  * External dependencies
  */
-var page = require( 'page' );
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-var config = require( 'config' ),
-	controller = require( 'my-sites/controller' ),
-	menusController = require( './controller' );
+import config from 'config';
+import { siteSelection, navigation, sites } from 'my-sites/controller';
+import { menus } from './controller';
 
-module.exports = function() {
+export default function() {
 	if ( config.isEnabled( 'manage/menus' ) ) {
-		page( '/menus/:site_id', controller.siteSelection, controller.navigation, menusController.menus );
-		page( '/menus', controller.siteSelection, controller.sites );
+		page( '/menus/:site_id', siteSelection, navigation, menus );
+		page( '/menus', siteSelection, sites );
```

Comment:
I don't think the `siteSelection` middleware is necessary in this route, since there is no `:site_id`, and the `sites` middleware takes care of showing the site selector.